packages/web-twig/src/Resources/components/Dropdown/README.md

```diff
@@ -0,0 +1,72 @@
+# Dropdown
+
+This is the Twig implementation of the [Dropdown] component.
+
+Basic example usage:
+
+```twig
+<Button data-toggle="dropdown" data-target="DropdownExample">Open Dropdown</Button>
+<Dropdown id="DropdownExample">Dropdown Content</Dropdown>
+```
+
+Usage with link:
+
+```twig
+<ButtonLink href="javascript:void(0)" data-toggle="dropdown" data-target="CollapseExample">Open Dropdown</ButtonLink>
+...
+```
+
+Responsive usage for tablet
+
+```twig
+...
+<Button class="d-tablet-none">Open Dropdown</Button>
+<Dropdown id="DropdownExample" breakpoint="tablet">Dropdown Content</Dropdown>
```

Comment:
This makes the trigger button disappear, right? However, the `breakpoint` feature is meant for turning off the full-width feature (which we forgot to implement, see above):

```suggestion
<Dropdown id="DropdownExample" fullWidth breakpoint="tablet">Dropdown Content</Dropdown>
```

---

Later (not in this PR) we can discuss a better API, e.g.:

```twig
<Dropdown id="DropdownExample" fullWidth fullWidthBreakpoint="tablet">Dropdown Content</Dropdown>
```

Or some kind of a more condensed syntax:

```twig
<Dropdown id="DropdownExample" fullWidthBreakpoint="all">Full-width on all breakpoints</Dropdown>
<Dropdown id="DropdownExample" fullWidthBreakpoint="tablet">Full-width UNTIL tablet breakpoint</Dropdown>
```

Reply:
Added as `isFullWidth`.